.gitignore

```diff
@@ -0,0 +1,7 @@
+node_modules
+.hintrc
+.stylelintrc.json
+package-lock.json
+package.json
+.github
```

Comment:
- [ ] Kindly remove these files here in the gitignore. This is the reason why the linters checks are not showing up on your pull request.
`ONLY THE test.md and node_modules are supposed to be ignored` :smile: